pyzo/pyzokernel/interpreter.py

```diff
@@ -437,13 +437,20 @@ def _run_startup_code(self, startup_info):
         
         if '\n' in script:
             # Run code later or now
-            firstline = script.split('\n')[0].replace(' ', '')
-            if firstline.startswith('#AFTER_GUI'):
-                self._codeToRunOnStartup = script
+            scriptSplit = script.split('\n')
+            foundAfter = False
+            for idx, line in enumerate(scriptSplit):
+                if line.replace(' ', '').startswith('#AFTER_GUI'):
```

Comment:
Small suggested tweak to make the code faster; the script is potentially very large.
```suggestion
                if line.startswith(('#AFTER_GUI', '# AFTER_GUI')):
```

Reply:
Oh wait, this is only the startup script, which we can assume to be small. Nevermind. I need a coffee.